src/settings/LocationLocations/LocationDetail.js

```diff
@@ -93,9 +93,8 @@ class LocationDetail extends React.Component {
     return (
       index === 0 ?
         <li key={index}>
-          {sp}
-          {' '}
-          (primary)
+          {/* eslint-disable-next-line react/jsx-one-expression-per-line */}
+          {sp} (primary)
```

Comment:
Will this help to solve the issue?
```suggestion
          <>{sp} (primary)</>
```

Reply:
Yes, thank you!